### PR TITLE
Con 2796 create version 5 legal agreement

### PR DESCRIPTION
**Pull request by Howesy77**

### CODE REVIEW

File: src/SFA.DAS.EmployerAccounts.Web/Extensions/AgreementTemplateExtensions.cs [hunk 1]
```diff
@@ -19,19 +20,22 @@ public static string InsetText(this AgreementTemplateViewModel agreementTemplate
...
+                case 5:
+                    bool v3orv4Accepted = GetV3orV4AgreementStatus(organisationAgreementViewModel);
+                    return v3orv4Accepted ? "This is a variation to the agreement we published 9 January 2020. You only need to accept it if you want to access incentive payments for hiring a new apprentice." : "This is a new agreement.";
                 default:
                     return string.Empty;
             }
         }
 
-        private static bool GetV3AgreementStatus(ICollection<OrganisationAgreementViewModel> organisationAgreementViewModel)
+        private static bool GetV3orV4AgreementStatus(ICollection<OrganisationAgreementViewModel> organisationAgreementViewModel)
```
Comment (cofaulco) [inline src/SFA.DAS.EmployerAccounts.Web/Extensions/AgreementTemplateExtensions.cs:31]: Next time there is another agreement will this become `GetV3OrV4OrV5AgreementStatus()`

**Review by cofaulco — COMMENTED** [review]
Looks fine, concerned about continuing forward in this manner if we continuously add new agreements

File: src/SFA.DAS.EmployerAccounts.Web/Extensions/AgreementTemplateExtensions.cs (hunk 1 above)
Comment (Howesy77) [reply src/SFA.DAS.EmployerAccounts.Web/Extensions/AgreementTemplateExtensions.cs]: Modified code

File: src/SFA.DAS.EmployerAccounts.Web/Extensions/AgreementTemplateExtensions.cs [hunk 2]
```diff
@@ -1,12 +1,12 @@
 ﻿using SFA.DAS.EmployerAccounts.Web.ViewModels;
 using System.Collections.Generic;
+using System.Linq;
 
 namespace SFA.DAS.EmployerAccounts.Web.Extensions
 {
     public static class AgreementTemplateExtensions
```
Comment (cofaulco) [inline src/SFA.DAS.EmployerAccounts.Web/Extensions/AgreementTemplateExtensions.cs:7]: Are there no unit tests for this extension method?

**Review by cofaulco — COMMENTED** [review]
Unit tests?

File: src/SFA.DAS.EmployerAccounts.Web/Extensions/AgreementTemplateExtensions.cs (hunk 2 above)
Comment (Howesy77) [reply src/SFA.DAS.EmployerAccounts.Web/Extensions/AgreementTemplateExtensions.cs:7]: Nope

**Review by cofaulco — APPROVED** [review]

**Pull request merged**